subprojects/shipkit/src/main/groovy/org/shipkit/internal/gradle/versionupgrade/UpgradeDependencyPlugin.java

```diff
@@ -231,6 +271,13 @@ public void execute(Task task) {
         });
     }
 
+    static String getCurrentVersionBranchName(String dependencyName, String version, String openPullRequestBranch) {
+        if (openPullRequestBranch != null) {
+            return openPullRequestBranch;
+        }
+        return getVersionBranchName(dependencyName, version);
```

Comment:
There is a caveat: it is very useful to have the version in the pull request title. However, when we update the PR, the title is no longer accurate. I don't want to block this PR but let's think how we can address it.